deploy_polygon/02_asset/02_deploy_asset_signed_auction_auth_v2.ts

```diff
@@ -0,0 +1,37 @@
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+import {DeployFunction} from 'hardhat-deploy/types';
+import {skipUnlessTest} from '../../utils/network';
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const {deployments, getNamedAccounts, getUnnamedAccounts} = hre;
+  const {deploy} = deployments;
+
+  const {
+    deployer,
+    assetAdmin,
+    assetAuctionFeeCollector,
+  } = await getNamedAccounts();
+  const others = await getUnnamedAccounts();
+
+  const assetContract = await deployments.get('PolygonAsset');
```

Comment:
Should be Asset on L1

Reply:
done -> https://github.com/thesandboxgame/sandbox-smart-contracts/pull/617/commits/b43663be68f4c7623e0dce34330ed7f653b25e52